pkg/autoupdate/findnew.go

```diff
@@ -251,8 +251,7 @@ func FindNewest(ctx context.Context, fullBinaryPath string, opts ...newestOption
 //
 // It makes some string assumptions about how things are named.
 func getUpdateDir(fullBinaryPath string) string {
-	installedPath := os.Getenv(LegacyAutoupdatePathEnvVar)
-	if installedPath != "" {
+	if installedPath := os.Getenv(LegacyAutoupdatePathEnvVar); installedPath != "" {
```

Comment:
Thank you. There's another one several lines down

Reply:
Pushed an update to remove the `strings.HasSuffix` check since `strings.TrimSuffix` is a no-op if the suffix isn't present -- unless you meant a different one and I'm missing it?

Reply:
https://github.com/kolide/launcher/blob/dec53eb992d5b3bcdc9ac605d78f17e25c3dc163/pkg/autoupdate/findnew.go#L334-L337